packages/devextreme/js/ui/stepper.js

```diff
@@ -3,3 +3,45 @@ import Stepper from '../__internal/ui/stepper/stepper';
 export default Stepper;
 
 // STYLE stepper
+
+/**
+ * @name dxStepperOptions.onContentReady
+ * @hidden
+ */
+
+/**
+ * @name dxStepperOptions.onItemHold
+ * @action
```

Comment:
Why on this line we declare @action but not for 'onContentReady' event? What is correct?